var/spack/repos/builtin/packages/conduit/package.py

```diff
@@ -256,6 +257,80 @@ def check_install(self):
             example = Executable("./conduit_example")
             example()
 
+    # TODO: Replace this method and its 'get' use for cmake path with
+    #   join_path(self.spec['cmake'].prefix.bin, 'cmake') once stand-alone
+    #   tests can access build dependencies through self.spec['cmake'].
+    def cmake_bin(self, set=True):
+        """(Hack) Set/get cmake dependency path."""
+        filepath = join_path(self.install_test_root, "cmake_bin_path.txt")
+        if set:
+            with open(filepath, "w") as out_file:
```

Comment:
Looks like `self.install_test_root` isn't getting created (anymore or in this case, not sure yet).  The following fix will resolve that issue:

```suggestion
        if set:
            mkdirp(os.path.dirname(filepath))
            with open(filepath, "w") as out_file:
```